var/spack/repos/builtin/packages/lammps/package.py

```diff
@@ -25,47 +25,152 @@ class Lammps(CMakePackage, CudaPackage):
 
     maintainers = ["rbberger"]
 
+    # rules for new versions and deprecation
+    # * new stable versions should be marked as preferred=True
+    # * a stable version that has updates and any of its outdated update releases should be
+    #   marked deprecated=True
+    # * only the latest update of any given stable release should be marked preferred=True
+    # * patch releases older than a stable release should be marked deprecated=True
     version("develop", branch="develop")
+    version("20221222", sha256="75372ee7ef982767fc4ed4dc95e20ddca8247419adeb0c1276c40e43d1eab955")
     version("20221103", sha256="d28517b84b157d4e46a1a64ed787b4662d8f2f5ade3f5a04bb0caed068f32f7e")
     version("20220915", sha256="392b8d35fc7919b0efaf8b389259a9b795a817e58e75e380467c63d03a0dab75")
     version("20220803", sha256="f37cb0b35c1682ffceae5826aadce47278aa7003099a1655fcea43acd7d37926")
     version(
-        "20220623.2", sha256="fdb5474135b17005030cd78ac85762fecf0944bdd27a81fbbe79b2b4522ccae7"
+        "20220623.2",
+        sha256="8a560213e83919623525c4a7c4b5f0eda35cdf3b0c0e6548fd891379e04ca9e6",
+        preferred=True,
     )
     version(
-        "20220623.1", sha256="1d4fb06a4621b271e8b54f31fef241d0569717a69c1b7646a2b76fe5ce47c966"
+        "20220623.1",
+        sha256="58e3b2b984f8935bb0db5631e143be2826c45ffd48844f7c394f36624a3e17a2",
+        deprecated=True,
+    )
+    version(
+        "20220623",
+        sha256="21533ce6f174c80815a48c99e5f3dd109e69d55c4cad47312d88a7190a35927f",
+        deprecated=True,
+    )
+    version(
+        "20220602",
+        sha256="3e8f54453e53b3b387a68317277f832b8cf64a981e64b21e98bb37ea36ac4a60",
+        deprecated=True,
+    )
+    version(
+        "20220504",
+        sha256="fe05bae8090fd0177b3c1b987cd32a9cb7cd05d790828ba954c764eb52e10b52",
+        deprecated=True,
+    )
+    version(
+        "20220324",
+        sha256="d791cc93eedfc345fdf87bfa5b6f7e17e461f86ba197f9e9c3d477ce8657a7ef",
+        deprecated=True,
+    )
+    version(
+        "20220217",
+        sha256="e5bd2bf325835fa98d1b95f0667c83076580916027df5b8109d5470d1b97da98",
+        deprecated=True,
+    )
+    version(
+        "20220107",
+        sha256="fbf6c6814968ae0d772d7b6783079ff4f249a8faeceb39992c344969e9f1edbb",
+        deprecated=True,
+    )
+    version(
+        "20211214",
+        sha256="9f7b1ee2394678c1a6baa2c158a62345680a952eee251783e3c246b3f12db4c9",
+        deprecated=True,
+    )
+    version(
+        "20211027",
+        sha256="c06f682fcf9d5921ca90c857a104e90fba0fe65decaac9732745e4da49281938",
+        deprecated=True,
+    )
+    version(
+        "20210929.3",
+        sha256="e4c274f0dc5fdedc43f2b365156653d1105197a116ff2bafe893523cdb22532e",
```

Comment:
I'm getting a different sha256 here:

```suggestion
        sha256="d7df30f7ce0bd8b7c21949cf5bdf1877858815379cddb8a264da7a8cbf699639",
```